quic/s2n-quic-core/src/crypto/tls.rs

```diff
@@ -237,3 +325,121 @@ impl HandshakeHeader {
 }
 
 s2n_codec::zerocopy_value_codec!(HandshakeHeader);
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use bolero::check;
+    use hex_literal::hex;
+
+    const MAX_HELLO_SIZE: u64 = if cfg!(kani) { 32 } else { 255 };
```

Comment:
This can't be the same as in `initial.rs`? (2 << 16)

Reply:
Initially it was, but I wanted to make it smaller here so it was easier to hit and get into the edge cases without creating a 64k buffer